[2.10] Backport native histograms documentation (#6757)
grafana/mimir: docs/sources/mimir/configure/configure-native-histograms-ingestion.md

# Configure native histograms

Prometheus native histograms ingestion is an **experimental** feature of Grafana Mimir.

You can configure native histograms ingestion via the Prometheus [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint globally or per tenant.

{{% admonition type="note" %}}
To enable support for querying native histograms together with [Grafana Mimir query sharding]({{< relref "../references/architecture/query-sharding" >}}), make sure that the flag `-query-frontend.query-result-response-format` is set to its default `protobuf` value on query frontends.
{{% /admonition %}}

## Configure native histograms globally

> To enable ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint for all tenants, set the flag `-ingester.native-histograms-ingestion-enabled=true` on ingesters.

Reviewer: ```suggestion
To enable ingesting Prometheus native histograms over the [remote write API]({{< relref "../references/http-api#remote-write" >}}) endpoint for all tenants, set the flag `-ingester.native-histograms-ingestion-enabled=true` on the ingesters.
```

> To limit the number of native histogram buckets per sample, set the `-validation.max-native-histogram-buckets` flag on distributors.

Reviewer: ```suggestion
To limit the number of native histogram buckets per sample, set the `-validation.max-native-histogram-buckets` flag on the distributors.
```

> The recommended value is 160 which is the default in the [OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/metrics/sdk/) for exponential histograms, which are a similar concept in OpenTelemetry.

Reviewer: ```suggestion
The recommended value is `160`, which is the default in the [OpenTelemetry SDK](https://opentelemetry.io/docs/specs/otel/metrics/sdk/) for exponential histograms. The native histogram buckets are similar in concept to exponential histograms in OpenTelemetry.
```
Spell out that last sentence. Please vet this for technical accuracy, because my change is a guess and has high potential for being inaccurate in this case.
